ppr-ui/tests/unit/test-data/mock-new-registration.ts

```diff
@@ -55,6 +55,16 @@ export const mockedMarriageMH: RegistrationTypeIF = {
   text: ''
 }
 
+export const mockedOtherCarbon: RegistrationTypeIF = {
```

Comment:
is there a reason for these mocked registration types other than getting the value we already have defined elsewhere? Could you grab this object from the list of registration types in resources instead? That way we don't have more than one place to change it if we need to alter it later

Reply:
We're using this in tests to set the store to test the other registration type changes... it needs to be a RegistrationTypeIF to set the store... if that makes sense

Reply:
done